src/Mobs/Path.h

```diff
@@ -0,0 +1,168 @@
+#pragma once
+
+#ifdef COMPILING_PATHFIND_DEBUGGER
+/* Note: the COMPILING_PATHFIND_DEBUGGER flag is used by Native/WiseOldMan95 to debug this class outside of MCServer.
+This preprocessor flag is never set when compiling MCServer. */
+#include "PathFinderIrrlicht_Head.h"
+#endif
+
+#include <vector>
+#include <queue>
+#include <unordered_map>
+using namespace std;
```

Comment:
This is problematic because its importing std::make_unique into the global namespace on systems with c++14 support which conflicts with our polyfill method in globals.h